examples/PutObjectUiProgressBar.java

```diff
@@ -86,17 +86,14 @@ private void uploadFile(String fileName) throws IOException, NoSuchAlgorithmExce
     // "YOUR-SECRETACCESSKEY");
 
     File file = new File(fileName);
-    BufferedInputStream bis;
-    try {
-      bis = new BufferedInputStream(new FileInputStream(file));
+    try (BufferedInputStream bis = new BufferedInputStream(new FileInputStream(file))) {
       ProgressMonitorInputStream pmis = new ProgressMonitorInputStream(
           this,
           "Uploading... " + file.getAbsolutePath(),
           bis);
 
       pmis.getProgressMonitor().setMillisToPopup(10);
       minioClient.putObject("bank", "my-objectname", pmis, bis.available(), "application/octet-stream");
-      pmis.close();
```

Comment:
So assuming if this is the behavior in Java 7 close is automatically called.

Reply:
Not java 7 behaviour, but as the link above describes, it closes what's declared in the try header